package.json

```diff
@@ -42,7 +42,7 @@
     "metalsmith": "^2.6.1",
     "metalsmith-beautify": "^0.2.0",
     "metalsmith-static": "0.0.5",
-    "node-fetch": "^2.6.7",
+    "node-fetch": "^3.3.2",
```

Comment:
I thought `node-fetch` 3.x wasn't compatible with `require` based JS and only worked in ES Modules:

https://github.com/node-fetch/node-fetch/blob/HEAD/docs/v3-UPGRADE-GUIDE.md#converted-to-es-module

> you are not able to import it with require.

Reply:
Oh, wait, I converted the Metalsmith plugins to modules ages ago....

Reply:
It's used in `index.mjs` like this

```js
import fetch from 'node-fetch';
```

I don't see require based usage in the project.